pkgs/development/tools/boost-build/default.nix

```diff
@@ -40,6 +40,9 @@ stdenv.mkDerivation {
   postPatch = ''
     substituteInPlace src/build-system.jam \
     --replace "default-toolset = darwin" "default-toolset = clang-darwin"
```

Comment:
```suggestion
      --replace-fail "default-toolset = darwin" "default-toolset = clang-darwin"
```

Reply:
Latest push drops this replacement because it fails with `--replace-fail`. It appears upstream has made clang the default.